### PR TITLE
[RDY] Fixed inconsistent naming of seeds (singular to plural)

### LINKED ISSUE #23027: Inconsistent naming of seed(s)
### Expected behavior
All ```seeds``` items should have only plural name since they represent grouped bunches of seeds, not single seeds.

### Actual behavior
Some of them named as 'seed' and other are 'seeds', for example, ```onion seed``` but ```datura seeds```. See the full list of seeds from core mod below:
seed

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
Fixes #23027.

# What was changed:

- Fixed inconsistent naming of seeds (singular to plural).

# Screenshots:

default

### CODE REVIEW

**Comment by illi-kun** [comment]
One thing we need to keep in mind that plural form of name for items without explicit definition of plural form is created by adding a "s" to the end of a name.

This is why we have records like that, to avoid the described behavior:
```
"name": "pair of boots",
"name_plural": "pairs of boots",
```

From other side, maybe it's not applicable to items with charges. Anyway, this is potential issue which has to be checked before merging this PR.

**Comment by ZhilkinSerg** [comment]
> One thing we need to keep in mind that plural form of name for items without explicit definition of plural form is created by adding a "s" to the end of a name.

Yes, I should return `name_plural`.

default

File: data/json/items/comestibles.json [hunk 1]
```diff
@@ -5624,7 +5624,7 @@
     {
         "type" : "COMESTIBLE",
         "id" : "marloss_seed",
-        "name" : "marloss seed",
+        "name" : "marloss seeds",
         "name_plural" : "marloss seeds",
```
Comment (Soyweiser) [inline data/json/items/comestibles.json]: As the marloss seed is described as ''This looks like a sunflower seed the size of your palm." perhaps it is an idea to keep this one singular? Esp as this is an alien type of seed.

**Comment by Soyweiser** [comment]
Great, now I can remove this issue from my list of inconsistencies that should be checked. (If it is committed I'll also update the wiki (esp as there have been other items moved/renamed which I still need to fix).

File: data/json/items/comestibles.json (hunk 1 above)
Comment (ZhilkinSerg) [reply data/json/items/comestibles.json]: Fair enough.

**Comment by Leland** [comment]
Now that's some quick issue fixin'

**Pull request merged**